README.md

```diff
@@ -31,13 +31,31 @@ DB_FIREBIRD_USERNAME=user
 DB_FIREBIRD_PASSWORD=password
 DB_FIREBIRD_CHARSET=ISO-8859-2
 ```
-If you're using models add
+
+### Usage
+#### Eloquent as model
+To override your default database connection define `$connection` name in your Eloquent Model
 ```php
+/**
+ * The connection name for the model.
+ *
+ * @var string
+ */
 protected $connection = 'myFirebirdConnection';
 ```
-to your Firebird models.
+After defining connection name you can use it in normal way as you use Eloquent:
+```php
+$user = User::where('id', 1)->get();
 
-If you're just running SQLs you can do:
+$users = User::all();
+```
+
+#### DB Query
+Each time you have to define connecion name, for example:
 ```php
-DB::connection('myFirebirdConnection')->select($sql, $bindings);
+$sql = 'SELECT * FROM USERS WHERE id = ?';
+$bindings = 1;
+$user = DB::connection('myFirebirdConnection')->select($sql, $bindings);
```

Comment:
First of all, `$bindings` has to be an array, not an integer. Second, I would rather give an example of writing SQLs with **named** parameters because they're easier to maintain and less prone to errors:
```php
DB::connection('myFirebirdConnection')->select('SELECT * FROM USERS WHERE id = :id', ['id' => 1])
```